### PR TITLE
104100 - Add possibility for online meeting

### PR DESCRIPTION
**Pull request by bratli**
Add flag to allow to create online meeting.

[AB#104100](https://statoildeveloper.visualstudio.com/b6d97460-2e53-47ae-aad4-b41396d90828/_workitems/edit/104100)

### CODE REVIEW

File: src/Equinor.ProCoSys.IPO.WebApi/Controllers/Invitation/CreateInvitationDto.cs [hunk 1]
```diff
@@ -16,5 +16,7 @@ public class CreateInvitationDto
         public IEnumerable<CreateParticipantDto> Participants { get; set; }
         public IEnumerable<string> McPkgScope { get; set; }
         public IEnumerable<string> CommPkgScope { get; set; }
+        public bool IsOnline { get; set; } = false;
+
```
Comment (janid1967) [inline src/Equinor.ProCoSys.IPO.WebApi/Controllers/Invitation/CreateInvitationDto.cs:20]: remove blank

File: src/tests/Equinor.ProCoSys.IPO.Command.Tests/InvitationCommands/CreateInvitation/CreateInvitationCommandHandlerTests.cs
Comment (janid1967) [inline src/tests/Equinor.ProCoSys.IPO.Command.Tests/InvitationCommands/CreateInvitation/CreateInvitationCommandHandlerTests.cs:1]: Add test to test online = true?

File: src/tests/Equinor.ProCoSys.IPO.Command.Tests/InvitationCommands/CreateInvitation/CreateInvitationCommandTests.cs [hunk 2]
```diff
@@ -51,7 +51,8 @@ public void Constructor_SetsProperties()
                 _type, 
                 _participants, 
                 _mcPkgScope,
-                null);
+                null,
+                false);
```
Comment (janid1967) [inline src/tests/Equinor.ProCoSys.IPO.Command.Tests/InvitationCommands/CreateInvitation/CreateInvitationCommandTests.cs:55]: true, and Assert that online is true

**Review by janid1967 — APPROVED** [review]
Approved with minor suggestions

**Review by TordJoranger — APPROVED** [review]

**Pull request merged**